pennylane/ops/op_math/decompositions/controlled_decompositions.py

```diff
@@ -497,16 +504,32 @@ def _mcx_many_borrowed_workers(wires, **kwargs):
 
 
 def _mcx_two_workers_condition(num_control_wires, num_work_wires, **__):
-    return num_control_wires > 2 and num_work_wires >= 2
+    return num_control_wires > 2 and (
+        num_work_wires >= 2 or (num_work_wires == 1 and num_control_wires < 6)
+    )
```

Comment:
This decomposition - despite its name - only uses a single work wire if `num_control_wires < 6`.